src/lib/encoder.ts

```diff
@@ -648,10 +690,11 @@ export const valueContentEncodingMap = (
       };
     }
     // https://github.com/lukso-network/LIPs/blob/master/LSPs/LSP-2-ERC725YJSONSchema.md#jsonurl
```

Comment:
This url could be replaced with the one on URI as in the docs it says this is deprecated
```suggestion
    // https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md#verifiableuri
```

Reply:
Good point. Also worth double checking where in the docs/ folder there are old references to JSONURL. We could also update the code snippets (or added new code snippets) for VerifiedURI.

Reply:
nice catch